### PR TITLE
Fixed some stuff

### PR DESCRIPTION
**Pull request by Waterrail**

### CODE REVIEW

File: assets/lang/sklmessages_pl_PL.properties [hunk 1]
```diff
@@ -80,8 +80,8 @@ editor.save=Zapisz profil
 
 settings.title=Ustawienia launchera
 settings.lang=J\u0119zyk
-settings.lang.help=Don't see your language?
-settings.lang.link=Help us translate!
+settings.lang.help=Widzisz błąd w tłumaczeniu?
+settings.lang.link=Pomóż go naprawić!
```
Comment (Greg-21) [inline assets/lang/sklmessages_pl_PL.properties]: "Don't see your language?" in Polish doesn't sound "Widzisz błąd w tłumaczeniu?" (and "Help us translate!" doesn't sound "Pomóż go naprawić!"). Is this a mistake, or maybe an announcement of future changes?

Comment (skmedix) [reply assets/lang/sklmessages_pl_PL.properties]: I'll remove these strings anyway, in next update, these strings will only appear when English is selected.

**Comment by skmedix** [comment]
@Greg-21 Today I finally manage to make it work without changing native letters to ASCII codes, so it's gonna work fine anyway 😄

**Comment by Greg-21** [comment]
@skmedix Oh, good job! So from now we don't have to worry about changing native letters to ASCII codes. Do the natives and ASCII codes work similarly, or maybe the ASCII codes work better, loads faster, etc.? If they work similarly, maybe should I convert all ASCII back to native letters in all lang files?

File: assets/lang/sklmessages_pl_PL.properties (hunk 1 above)
Comment (Greg-21) [reply assets/lang/sklmessages_pl_PL.properties]: OK. Thanks for your reply.

**Comment by skmedix** [comment]
Now it doesn't matter. If you want you can convert them all. The problem was in JDK8, as it reads files by default in wrong encoding, and it was a lil bit hard to override. But I found a workaround 😄

**Pull request merged**